plugins/org.jkiss.dbeaver.ext.gaussdb/plugin.xml

```diff
@@ -33,7 +33,7 @@
                         defaultPort="8000"
                         description="%driver.gaussdb.description"
                         categories="sql">
-                    <file type="jar" path="https://repo.huaweicloud.com/repository/maven/huaweicloudsdk/gsjdbc4/gsjdbc4/1.1/gsjdbc4-1.1.jar" bundle="!drivers.gaussdb"/>
+                    <file type="jar" path="https://repo.huaweicloud.com/repository/maven/huaweicloudsdk/gsjdbc4/gsjdbc4/1.2/gsjdbc4-1.2.jar" bundle="!drivers.gaussdb"/>
```

Comment:
Can we use Maven artifact reference (+ custom Maven repo - see example in Cubrid extension)?
Thus we will always have the latest driver version

Reply:
Okay, I'll take care of the next version.